float64 read_time
uint8 id
uint8 mode
float32 x
float32 y
float32 strength
float32 azimuth